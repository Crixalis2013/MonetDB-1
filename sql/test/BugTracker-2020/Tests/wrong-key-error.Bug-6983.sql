start transaction;
create table a (id int primary key);
insert into a values (1);

create table b (id int primary key);

create table c (a int references a(id), b int references b(id));
insert into c (a, b) values (1, 2); -- 40002!INSERT INTO: FOREIGN KEY constraint 'c.c_b_fkey' violated
rollback;
