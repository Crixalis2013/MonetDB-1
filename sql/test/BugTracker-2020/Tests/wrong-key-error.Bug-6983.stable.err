stderr of test 'wrong-key-error.Bug-6983` in directory 'sql/test/BugTracker-2020` itself:


# 15:23:21 >  
# 15:23:21 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-3778" "--port=36467"
# 15:23:21 >  

MAPI  = (monetdb) /var/tmp/mtest-3778/.s.monetdb.36467
QUERY = insert into c (a, b) values (1, 2); -- 40002!INSERT INTO: FOREIGN KEY constraint 'c.c_b_fkey' violated
ERROR = !INSERT INTO: FOREIGN KEY constraint 'c.c_b_fkey' violated
CODE  = 40002

# 15:23:21 >  
# 15:23:21 >  "Done."
# 15:23:21 >  

