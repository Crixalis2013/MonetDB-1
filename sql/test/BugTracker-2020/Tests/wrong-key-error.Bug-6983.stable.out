stdout of test 'wrong-key-error.Bug-6983` in directory 'sql/test/BugTracker-2020` itself:


# 15:23:21 >  
# 15:23:21 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-3778" "--port=36467"
# 15:23:21 >  

#start transaction;
#create table a (id int primary key);
#insert into a values (1);
[ 1	]
#create table b (id int primary key);
#create table c (a int references a(id), b int references b(id));
#rollback;

# 15:23:21 >  
# 15:23:21 >  "Done."
# 15:23:21 >  

